readme/configuration-management.md

```diff
@@ -70,14 +70,39 @@ We need to find a better way of preventing this than manually monitoring module
 
 ## Configuration Split workflow
 
-BLT recommends using the Config Split module to manage configuration on most projects. For a description of how to use Config Split, see this excellent [blog post by Jeff Geerling](https://www.jeffgeerling.com/blog/2017/adding-configuration-split-drupal-site-using-blt-and-acquia-cloud).
+### Overview
+
+BLT recommends using the Config Split module to manage configuration on most projects. For an overview of how to use Config Split, see this excellent [blog post by Jeff Geerling](https://www.jeffgeerling.com/blog/2017/adding-configuration-split-drupal-site-using-blt-and-acquia-cloud).
```

Comment:
Yeah I definitely think we should default to config_split. An update hook (or just a line in the release notes) for existing features-based projects sounds good to me.